### PR TITLE
server/perkeepd/ui: Indent using tabs

### PR DESCRIPTION
**Pull request by omarchehab98**
Convert all files to tabs instead of spaces to adhere with style guide

### CODE REVIEW

**Comment by bradfitz** [comment]
Let's not modify svg files.

Which style guide, btw? Link it in commit message?

Also, you can delete the Change-Id. That's no longer needed. What added that? Did you do it by hand? Or you're manually moving over a change from Gerrit?

**Comment by omarchehab98** [comment]
Sure, I'll revert the SVG files.

Whoops, should've mentioned that it is the styleguide found in the doc
folder doc/web-ui-styleguide.md

Change-Id was added automatically by a commit hook, I should remove the
hook since it's not longer needed.

**Comment by bradfitz** [comment]
@googlebot, PTAL.

**Comment by omarchehab98** [comment]
@bradfitz ping

File: server/perkeepd/ui/mobile.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
 		<script>
 			var render = function() {
 				var newURL = new goog.Uri(window.location.href);
-			        var MobileSetupView = React.createFactory(cam.MobileSetupView);
+					var MobileSetupView = React.createFactory(cam.MobileSetupView);
```
Comment (mpl) [inline server/perkeepd/ui/mobile.html]: that looks wrong

**Pull request closed without merging**